ConnectionFactory.php

```diff
@@ -86,6 +87,11 @@ public function createConnection(array $params, ?Configuration $config = null, ?
                        after dropping support for doctrine/dbal 2, since this
                        package requires doctrine/dbal 3.3.2 or higher. */
                     if (isset($params['defaultTableOptions']['collate']) && defined('Doctrine\DBAL\Connection::PARAM_ASCII_STR_ARRAY')) {
+                        Deprecation::trigger(
+                            'doctrine/doctrine-bundle',
+                            'https://github.com/doctrine/DoctrineBundle/pull/1478',
```

Comment:
Wouldn't it be better if this was link to DBAL where they changed from collate to collation?

Reply:
I think you are right, since here `defaultTableOptions` is only validated as an array, and there is no validation of the keys. Let me change that.